src/test/test-random-util.c

```diff
@@ -51,6 +55,50 @@ static void test_rdrand(void) {
         }
 }
 
+#define TOTAL 100000
+
+static void test_random_u64_range_one(unsigned mod) {
+        log_info("/* %s(%u) */", __func__, mod);
+
+        unsigned max = 0, count[mod];
```

Comment:
if you use a VLA, maybe put an `assert(mod < $SOME_LARGE_VALUE)` first to clarify this is not going to overrun the stack. static checkers might care as might casual onlookers

Reply:
It's hardcoded... I assumed I can put 400k on the stack in test code.